.github/workflows/format.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: 👔 Format
         run: npm run format --if-present
 
+      - name: 🔗 Convert Docs links to references
```

Comment:
I think we should do this before the `format` step, as all JS/TS in our MD will be in the wrong format otherwise

Reply:
That's correct, but it's using `markdown` for everything it seems.
Even if we have code examples in there.

Don't know how to fix it in another way though. 🤔